com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -271,19 +271,18 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
 
             // When post-processing is enabled we can use the stack to resolve rendering to camera target (screen or RT).
             // However when there are render passes executing after post we avoid resolving to screen so rendering continues (before sRGBConvertion etc)
-            bool dontResolvePostProcessingToCameraTarget = hasCaptureActions || hasPassesAfterPostProcessing || applyFinalPostProcessing;
+            bool resolvePostProcessingToCameraTarget = !hasCaptureActions && !hasPassesAfterPostProcessing && !applyFinalPostProcessing;
 
             if (lastCameraInTheStack)
             {
                 // Post-processing will resolve to final target. No need for final blit pass.
                 if (applyPostProcessing)
                 {
-                    var destination = dontResolvePostProcessingToCameraTarget ? m_AfterPostProcessColor : RenderTargetHandle.CameraTarget;
+                    var destination = resolvePostProcessingToCameraTarget ? RenderTargetHandle.CameraTarget : m_AfterPostProcessColor;
 
                     // if resolving to screen we need to be able to perform sRGBConvertion in post-processing if necessary
-                    bool doSRGBConvertion = !(dontResolvePostProcessingToCameraTarget || (m_ActiveCameraColorAttachment != RenderTargetHandle.CameraTarget));
+                    bool doSRGBConvertion = resolvePostProcessingToCameraTarget;
```

Comment:
Are we sure we don't need the m_ActiveCameraColorAttachment check?
```suggestion
                    bool doSRGBConvertion = resolvePostProcessingToCameraTarget && m_ActiveCameraColorAttachment == RenderTargetHandle.CameraTarget;
```

Reply:
I'm not sure, why it even was there in the first place. m_ActiveCameraColorAttachment is either backbuffer or temporary RT, depending on whether we need a temporary RT or not, and describes the source for PostFX. We need to do the conversion if destination of PostFX is the backbuffer. Destination of PostFX depends only on resolvePostProcessingToCameraTarget.

Reply:
I'm adding @Unity-Technologies/gfx-qa-foundation to test camera stacking and Post, otherwise seems good.